### PR TITLE
Add gnps_id to molecular plotting return values

### PR DESCRIPTION
**Pull request by bernardolk**
So the user of the function can know which molecules each plot is for (not all are plotted, some are filtered in the validation step)
Before we returned just a list of images in the first tuple value, now its a dict of gnps_id: img

### CODE REVIEW

**Review by cereniyim — APPROVED** [review]
Other PRs from you will fix CI right?

File: omigami/plotting.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ def plot_molecule_structure_grid(
         draw_indices: bool = False,
         img_size: Tuple[int, int] = (200, 200),
         substructure_highlight: str = "",
-    ) -> (List[PngImageFile], List[str]):
+    ) -> (Dict[str, PngImageFile], List[str]):
```
Comment (carloszanella) [inline omigami/plotting.py:40]: what are these parenthesis for? should it be a `Tuple`?

Comment (carloszanella) [reply omigami/plotting.py:40]: looks like it from return value

**Review by carloszanella — COMMENTED** [review]
just one small comment

**Pull request merged**